caiman/motion_correction.py

```diff
@@ -2829,14 +2837,14 @@ def motion_correct_batch_rigid(fname, max_shifts, dview=None, splits=56, num_spl
         if isinstance(fname, tuple):
             base_name=os.path.split(fname[0])[-1][:-4] + '_rig_'
         else:
-            base_name=os.path.split(fname)[-1][:-4] + '_rig_'
+            base_name=os.path.split(fname)[-1][:-4] + '_rig_'   
```

Comment:
```suggestion
            base_name=os.path.split(fname)[-1][:-4] + '_rig_'
```

Reply:
Don't leave whitespace at the end of lines.

Reply:
Thank you @jverswijver!